tailscale/client.go

```diff
@@ -937,6 +937,160 @@ func (c *Client) SetDeviceIPv4Address(ctx context.Context, deviceID string, ipv4
 	return c.performRequest(req, nil)
 }
 
+const (
+	EmptyProviderType      ProviderType = ""
```

Comment:
Hmm were you thinking of prefixing with something like `Webhook` for `WebhookEmptyProviderType` for this first entry as an example?

Reply:
Exactly. It gets pretty wordy though, which is why I was wondering whether using a separate package would make sense. At some point, this client package is going to get ginormous, so at a minimum it would be nice to at least move each main entity to its own file (though package would be better I think).

Reply:
Yeah that is fair! Definitely agree RE: package / file size, think splitting entities into separate files at the minimum makes sense. 

Is splitting off into separate packages going to be odd UX in the short term? e.g., since the existing client code / endpoints will be in `tailscale` vs newer logic ending up in separate packages.